### DISCUSSION TITLE
Problems with image postgres:16

### DISCUSSION THREAD

**Discussion by iqapps** [immich-app/immich#4337] [Q&A]
Hi,

I run Immich on Synology with Portainer, and was recently having problems starting Immich after an image update, because the image postgres:16 would not start with the immich data, saying that version 16 is not backward compatible with v. 15 dataformats.

I solved this by specifying the container to use the postgres:15 image.

Is there work in progress on this topic, or is it just me having this problem ? (and if so why)

**Comment by bo0tzz** [comment]
Postgres needs manual intervention for major version upgrades. https://www.postgresql.org/docs/current/upgrading.html

**Comment by iqapps** [comment]
As postgres needs manual intervention, should it not be considered to add a migration job to upgrade postgres data.

**Reply by bo0tzz** [reply]
No, that is not the task of an application like Immich
